src/components/JumpIn/JumpIn.tsx

```diff
@@ -1,27 +1,23 @@
 import * as React from 'react'
 import { Props } from './JumpIn.types'
 import { getSelection, getCenter, getExplorerURL as getLandURL } from 'modules/land/utils'
-import { getExplorerURL as getCollectionURL } from 'modules/collection/utils'
 import './JumpIn.css'
 
 export default class JumpIn extends React.PureComponent<Props> {
-  render() {
-    const { size = 'medium', land, collection, items } = this.props
-
-    let url = ''
-    if (items) {
-      url = getCollectionURL({ item_ids: items.map(item => item.id) })
-    } else if (collection) {
-      url = getCollectionURL({ collection })
-    } else if (land) {
+  jumpToLand = () => {
+    const { land } = this.props
+    if (land) {
       const selection = getSelection(land)
       const [x, y] = getCenter(selection)
-      url = getLandURL(x, y)
-    } else {
-      throw new Error('You need to supply at least one of the available props')
+      const newWindow = window.open(getLandURL(x, y), '_blank,noreferrer')
+      if (newWindow) {
+        newWindow.focus()
```

Comment:
Why this?

Reply:
I copied the same functionality from the `JumpIn` button, but it seems that it's not required, the window gets focused automatically.